packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx

```diff
@@ -15,13 +15,13 @@ export const Appearance = () => {
   const styles = useContainerStyles();
   return (
     <div className={styles.container}>
-      <InteractionTag icon={<CalendarMonth />} dismissible>
+      <InteractionTag icon={<CalendarMonthRegular />} dismissible>
```

Comment:
why revert back to unbundled icons here?

Reply:
oh I see it's only outline appearance tags affected

Reply:
> I'd still keep using bundled icons IMO, there's not much consequence to and it's what we encourage partners to do

Oh I see, I'll use bundled icon for the stories that I'm already bundle them :)